.github/workflows/build_wheels.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Get tag
         id: get_tag
         run: |
-          echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+          echo "TAG=v2.60.0-RC0" >> $GITHUB_OUTPUT
```

Comment:
We probably want this one change to be in a separate PR entirely, and then just run the workflow(s) from a branch with this one value changed, instead of running them from master.